test/typed_buffers_test.dart

```diff
@@ -185,35 +190,35 @@ void testInt(int bits, buffer(int length), {String testOn}) {
 
 const List<int> intSamples = const [
   0x10000000000000001,
-  0x10000000000000000,  // 2^64
+  0x10000000000000000, // 2^64
```

Comment:
The first two literals are too big for Dart 2 anyway.
Is it worth it to remove them now? (Probably not yet, the SDK constraint is still "<2.0.0").

Reply:
Let's wait...